File: app/src/main/java/ir/touristland/Models/CityItem.java
package ir.touristland.Models;

import java.io.Serializable;

public class CityItem implements Serializable {

    public static final long serialVersionUID = 1L;
    public String _Code = "";
    public String _city_name_fa = "";

    public String toString() {
        return (_city_name_fa);
    }

    public String getCityNameFa() {
        return _city_name_fa;
    }

    public void setCityNameFa(String city_name_fa) {
        _city_name_fa = city_name_fa;
    }

    public String getCode() {
        return _Code;
    }

    public void setCode(String Code) {
        _Code = Code;
    }

}

File: app/src/main/java/ir/touristland/Models/CityItem.kt
package ir.touristland.Models

import java.io.Serializable

class CityItem : Serializable {
    var code = ""
    var cityNameFa = ""

    override fun toString(): String {
        return cityNameFa
    }

    companion object {

        const val serialVersionUID = 1L
    }

}
